src/sagemaker/automl/automl.py

```diff
@@ -45,6 +45,7 @@ def __init__(
         self,
         inputs,
         target_attribute_name,
+        sample_weight_attribute_name=None,
```

Comment:
Generally its not recommended to have a new argument added in the middle for backwards compatibility reasons as it can break the existing code of our customers using this functionality without keyword arguments, can we move this argument to the end?